tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1501,10 +1501,11 @@ def decode_image(contents, channels=None, name=None):
     contents: 0-D `string`. The encoded image bytes.
     channels: An optional `int`. Defaults to `0`. Number of color channels for
       the decoded image.
+    dtype: An optional `tf.DType` from: `tf.uint8, tf.uint16`. Defaults to `tf.uint8`.
```

Comment:
How about phrasing as:

```python
dtype: The desired `tf.DType` of the returned `Tensor`. Must be either `tf.uint8` or `tf.uint16`.
```

Also, it's probably worth ensuring that the expectation is met by adding a:
```python
if dtype != tf.uint8 and dtype != tf.uint16:
  raise ValueError('dtype must be either tf.uint8 or tf.uint16')
```
to the body.